src/style/core.ts

```diff
@@ -29,6 +31,7 @@ const darkTheme = `
   --rdg-row-hover-background-color: hsl(0deg 0% 9%);
   --rdg-row-selected-background-color: hsl(207deg 76% 42%);
   --rdg-row-selected-hover-background-color: hsl(207deg 76% 38%);
+  --rdg-cell-frozen-box-shadow: calc(2px * var(--rdg-sign)) 0 5px -2px rgba(136, 136, 136, 0.3);
```

Comment:
We don't need to define this twice if it has the same value. We can move it next to `--rdg-selection-color` and `--rdg-font-size`